### DISCUSSION TITLE
Forks

### DISCUSSION THREAD

**Discussion by mcguirepr89** [mcguirepr89/BirdNET-Pi#675] [Show and tell]
### Show it off!! :loudspeaker: :champagne: 
# Let the hunt for a better BirdNET-Pi begin!!!! 

I hope this thread can be used to help folks document anything special or different about their fork of this repo. 

If you think a fork is cool, be sure to up-vote it so that it can stand out!
Happy birding ~~

**Comment by Fish31520** [comment]
@mcguirepr89 Sorry, I'm just using it "Vanilla", no amazing fork for me, just the regular software. 😅

Have a good quest ! 😄

**Comment by Svardsten53** [comment]
Please see what was written here: https://github.com/mcguirepr89/BirdNET-Pi/discussions/69#discussioncomment-4462067

**Comment by Svardsten53** [comment]
I hope we can continue to discuss here how we can develop BirdNET-Pi in the future as well.

I joined almost from the beginning and have made my own adaptations and also translated into Swedish and everything now works very well both for me and other installations in Sweden, among them at least two professional bird observatories. The program has many good functions and works with very high uptime, and as I see it, no improvements are needed regarding this.

What can be improved is the analysis part. The TFLite model unfortunately has some flaws and gives a lot of false hits. It would be best if someone could implement the new model in the existing version of BirdNET-Pi, but apparently this is not easy.

Would it be possible as an alternative to do a filtering of the analysis result using eBird, similar to what is now done in BirdWeather, where completely incorrect results are filtered out based on previous findings on the location at that time of year? Perhaps one could then lower the confidence value to include more findings and then remove those that are incorrect. The choice of whether to use eBird or not could be added to the settings of BirdNET-Pi.

Unfortunately I have no programming skills myself in this area, but I'm sure others could do this and that it's easier than implementing the new model.

**Reply by krummrey** [reply]
Having a two stage model is what I had in mind. If the the new model is too heavy for a raspberry pi, then use the old model with a lower detection value. In a  second step run only the detections through the new model on a different machine once in a while to validate the detections. 
Not as elegant as having it all on one machine, but most of us have a laptop or PC that has plenty of power for the second step.
I also lack the proficiency in a language to come up with a system like that, but maybe someone does.

**Reply by Svardsten53** [reply]
I guess the new model can't be so heavy that it can't run in a newer Raspberry Pi. Today's 15 seconds sound files are analyzed in a few seconds and the CPU load is not very high. Since I prefer the result directly on the web page, I am not particularly interested in having to do manual work afterwards. If the new model cannot be implemented, a sanity check with eBird should be an alternative for the time being if possible.

**Comment by lloydbayley** [comment]
Can thoroughly recommend https://github.com/Nachtzuster/BirdNET-Pi
I'm running it and all problems with notifications etc are fixed and the system is working well.
It's also been updated for Bookworm AND it will install on an x86 system (I've tested this too).

**Reply by BAM-Team** [reply]
Looks very cool...I will try this out :-D Thanks for your posting
Best wishes, Carsten

**Reply by BAM-Team** [reply]
I run some orangepi02w with the Nachtzuster repo and can recommend it and new features (weekly report) and ideas come very quickly. Also there are some performance improvements "under the hood" 
It is excellent for mobile stations etc. With BAM-Team I run 3 monitoring stations on Patricks repo because of the stability and comparability. But maybe, I will switch completely to nachtzuster repo somedays...
